test/distrib/bazel/test_single_bazel_version.sh

```diff
@@ -51,12 +51,14 @@ FAILED_TESTS=""
 export OVERRIDE_BAZEL_VERSION="$VERSION"
 # when running under bazel docker image, the workspace is read only.
 export OVERRIDE_BAZEL_WRAPPER_DOWNLOAD_DIR=/tmp
+
 bazel build -- //... "${EXCLUDED_TARGETS[@]}" || FAILED_TESTS="${FAILED_TESTS}Build "
```

Comment:
did you forget to change to `tools/bazel` as well?

Reply:
Done.